pytd/client.py

```diff
@@ -202,7 +202,7 @@ def query(self, query, engine=None, **kwargs):
             - ``wait_callback`` (function): called every interval against job itself
             - ``engine_version`` (str): run query with Hive 2 if this parameter
               is set to ``"experimental"`` and ``engine`` denotes Hive.
-              https://tddocs.atlassian.net/wiki/spaces/PD/pages/1083123/Using+Hive+2+to+Create+Queries
+              https://docs.treasuredata.com/display/public/PD/Using+Hive+2
```

Comment:
I think this might be better place to change?

```suggestion
              https://docs.treasuredata.com/display/public/PD/Writing+Hive+Queries
```

Reply:
Overlooked and didn't notice the same page exists in the new site... 253585cf7e1cdf79acfa8bf5a4bbce60b2342fff